_episodes/03-intro-git-github.md

```diff
@@ -1474,14 +1476,40 @@ origin   https://github.com/USERNAME/un-report.git (fetch)
 ```
 {: .output}
 
-Once the remote is set up, this command will push the changes from
-our local repository to the repository on GitHub:
+Now we want to send our local git information to GitHub. While the default for code you put on GitHub is that anyone can view or make copies of your code, in order to make changes to your repository, you need to be able to log in so GitHub can recognize you as someone who is authorized to make changes.
+
+### Setting up your GitHub Personal Access Token (PAT)
+
+When you use the GitHub website, you need to login with a username and password. By default, only you will be able to make any changes to the repositories you create. In order to perform git commands on your own computer that interact with GitHub, we need a way to tell GitHub who you are. Rather than requiring you to type your password every time, you can create identify yourself with a personal access token (PAT). Let's first tell git that we would like it to remember our credentials so we don't have to constantly retype them. At the command line type:
+
+```
+git config --global credential.helper store
+```
+{: .language-bash}
+
+Like the previous `git config` commands we ran before, this tells git to store our account information so it doesn’t have to ask us for it every time time we use git on the command line.
+
+The information git stores is your personal access token (PAT). These tokens are basically a secret word that only you know that allows you to access all your stuff. Think of these tokens like a key to your house. You never want to hand over the keys to your house to someone you don't trust. But as long as you hang on to that key, you are free to access all your stuff.
```

Comment:
With this analogy, learners might wonder how PATs are different from passwords (they're computer-generated rather than human-generated, and PATs aren't meant to be remembered by humans) and why GitHub wants us to use PATs instead of passwords (they're more secure). Perhaps a quick/short explanation right at the end of this analogy would be useful? Let me know what you think @MrFlick!

Reply:
Good suggestion. I added a callout box for this topic.